helpdesk/timesheet_and_invoice/invoice_time.rst

```diff
@@ -1,126 +1,111 @@
-===================================
-Record and Invoice Time for Tickets
-===================================
+========================================================
+Invoice Time Spent on Tickets (Prepaid Support Services)
+========================================================
 
-You may have service contracts with your clients to provide them
-assistance in case of a problem. For this purpose, Odoo will help you
-record the time spent fixing the issue and most importantly, to invoice
-it to your clients.
+Have the option to work with prepaid support services, meaning that a sales order and a
+corresponding invoice are issued and, once the service is done, you can deduct the time spent. Odoo
+allows it to happen because the applications are fully integrated, resulting in faster responses
+to your customer needs.
 
-The modules needed
-==================
+Set up
+======
 
-In order to record and invoice time for tickets, the following modules
-are needed : Helpdesk, Project, Timesheets, Sales. If you are missing
-one of them, go to the Apps module, search for it and then click on
-*Install*.
+Step 1: Set up a helpdesk team
+------------------------------
 
-Get started to offer the helpdesk service
-=========================================
+| Go to :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams`, create or edit an existing
+  team, and enable *Timesheet on Ticket* and *Time Reinvoicing*.
+| Select or create a project under *Timesheet on Ticket*. The selected/created is
+  the one at which employees timesheet on by default. However, it can be ultimately modified on
+  each ticket.
 
-Step 1 : start a helpdesk project
----------------------------------
-
-To start a dedicated project for the helpdesk service, first go to
-:menuselection:`Project --> Configuration --> Settings` and make sure
-that the *Timesheets* feature is activated.
-
-.. image:: media/invoice_time01.png
+.. image:: media/sell_track_hours.png
    :align: center
+   :alt: View of a helpdesk team settings page emphasizing the timesheet on ticket and time
+         reinvoicing features in Odoo Helpdesk
 
-Then, go to your dashboard, create the new project and allow timesheets
-for it.
+Step 2: Set up a service
+------------------------
 
-Step 2 : gather a helpdesk team
--------------------------------
+| Go to :menuselection:`Sales --> Configuration --> Settings` and enable *Units of Measure* to
```

Comment:
Just thought about it, but maybe it would be interesting to link to the Units of Measures doc, no? :)

Reply:
yeah..why not?! added in the seealso 'cause it is super duper optional (: